src/wizard/wizard-dialog.ts

```diff
@@ -119,8 +119,11 @@ export class ESPHomeWizardDialog extends LitElement {
       [heading, content, hideActions] = this._renderAskESPHomeWeb();
     } else if (this._state === "basic_config") {
       [heading, content, hideActions] = this._renderBasicConfig();
-    } else if (this._state === "pick_board") {
+    } else if (this._state === "pick_platform") {
       heading = "Select your device type";
+      content = this._renderPickPlatform();
+    } else if (this._state === "pick_board") {
+      heading = "Select your board";
```

Comment:
If it's not Raspberry Pi, include the platform in the name.

> Select your ESP32-S3 board

Reply:
Added a `showPickerTitle` property to the platform list, determining if the platform name should be visible in the board picker.